app/src/main/java/com/apps/adrcotfas/goodtime/Preferences.java

```diff
@@ -38,19 +38,19 @@ public Preferences(SharedPreferences pref) {
     }
 
     public int getSessionDuration() {
-        return mPref.getInt(SESSION_DURATION, 25);
```

Comment:
We can't access the resources like this from here, so I'll revert these 4 changes too.
Running the app for the first time will result in "0" session duration.